### ISSUE TITLE
.width(value) sets incorrect width value if block has border and value specified in ems and box-sizing:border-box

### ISSUE THREAD

**Issue by mzgol** [jquery/jquery#1712]
Originally reported by tde: http://bugs.jquery.com/ticket/14038

API documentation specifies: "Note that .width("value") sets the content width of the box regardless of the value of the CSS box-sizing property."

There are two key factors leading to this bug

"box-sizing:border-box"
"border:1px solid black"
width specified in ems
Code that reproduces this bug

```html
<html>
	<head>
		<style>
			#container {width:100%;}
			#box {border:1px solid black; margin:0 auto; box-sizing:border-box;}
		</style>
		<script type="text/javascript" src="http://code.jquery.com/jquery-2.0.1.js"></script>
		<script>
			var $ = jQuery.noConflict();
			$(document).ready(function() {
				$('#box').width('10em'); // width should be 10em, but jquery set it to 12em
			});
		</script>
	</head>
	<body>
		<div id="container">
			<div id="box">
				<p>some text</p>
			</div>
		</div>
	</body>
</html>
```

**Comment by Gerst20051** [comment]
:+1: just ran into this bug (is it a bug or expected behavior?) but as pointed out above setting the width using ```.css('width', width);``` works as expected.

**Comment by mzgol** [comment]
@Gerst20051 We've accepted it as a bug and (currently) plan to fix it for 3.0.0. We still [have a lot to do](https://github.com/jquery/jquery/milestones/3.0.0) for 3.0.0 though!
